ofono/include/sms-filter.h

```diff
@@ -95,7 +95,7 @@ typedef void (*ofono_sms_filter_send_datagram_cb_t)
 				const struct ofono_sms_address *addr,
 				int dst_port, int src_port,
 				const unsigned char *buf, unsigned int len,
-				void *data);
+				int flags, void *data);
```

Comment:
This is an interface, it can only be changed in a backward compatible manner.

Reply:
Technically yes but this wasn't ever used as there was no implementation for it before. It did exist only in the headers. Well, filter does perhaps need the flags?

Reply:
Let's not touch the filtering interface, If certain plugin would actually need more info, the interface can be extended with another callback. Until then, I don't think it would be worth the trouble.

Reply:
Dropped: a31e99b0a6bb56b996c9d1e9f3bd7cf0aef2f5a0

Reply:
Were these drops lost in some squash?